config/locales/forms/en.yml

```diff
@@ -137,5 +137,5 @@ en:
       intro: Add a security key that meets the FIDO standard as your authentication
         method. You can add as many security keys as you want. To get started,
         first give your security key a nickname.
-      login_text: When you are ready to authenticate, press the button
+      login_text: 'When you are ready to authenticate, press the button:'
```

Comment:
We have two different texts for security key vs. platform authenticators. Why is that? 🤔 

>When you are ready, press the button:

>When you are ready to authenticate, press the button:

Reply:
I'm not sure, especially since there isn't necessarily a button on platform auth to begin with?

Reply:
> I'm not sure, especially since there isn't necessarily a button on platform auth to begin with?

I think in this case, the "button" is referring to the one on the page that the user clicks to initiate the authentication:

![image](https://user-images.githubusercontent.com/1779930/151200945-f269a11f-e2ea-47c9-b72c-2114ded2f2c8.png)